tests/conftest.py

```diff
@@ -26,6 +27,8 @@
 from pennylane.devices import DefaultGaussian
 from pennylane.operation import disable_new_opmath_cm, enable_new_opmath_cm
 
+sys.path.append(os.path.join(os.path.dirname(__file__), "helpers"))
```

Comment:
I'm not sure why, but I think this may be the culprit for the failing tests.

Reply:
Yes, I changed the module name.